group-coordinator/src/main/resources/common/message/ConsumerGroupRegularExpressionKey.json

```diff
@@ -0,0 +1,28 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// KIP-848 is in development. This schema is subject to non-backwards-compatible changes.
+{
+  "type": "data",
+  "name": "ConsumerGroupRegularExpressionKey",
+  "validVersions": "15",
```

Comment:
I had used 16 here because there is another record key already using 15 https://github.com/apache/kafka/blob/8cbd2edfe782997380683cbfa7451a4f2de893f0/group-coordinator/src/main/resources/common/message/ShareGroupStatePartitionMetadataKey.json#L20
should it be 16?

Reply:
oh, i see. i did not see it because it is not in the used in the replay method yet. let me change it.